local/gugcat/renderer.php

```diff
@@ -200,8 +200,9 @@ public function display_aggregation_tool($rows, $activities) {
                         button.click();
                         "></i></td>';
             $htmlrows .= html_writer::tag('td', $row->completed);
+            $zero_resit_weight = $row->grdedresit == 1 ? ' class = "zero-resit-weight"': '';
             $htmlrows .= ($row->aggregatedgrade->display != get_string('missinggrade', 'local_gugcat')) 
-            ? html_writer::start_tag('td').$row->aggregatedgrade->display.$this->context_actions($row->studentno, null, true, str_replace('_cnum', $row->cnum, $gradeformurl), true).html_writer::end_tag('td')
+            ? html_writer::start_tag('td'.$zero_resit_weight).$row->aggregatedgrade->display.$this->context_actions($row->studentno, null, true, str_replace('_cnum', $row->cnum, $gradeformurl), true).html_writer::end_tag('td')
```

Comment:
Does this one does not produce an error? You're concatenating it to the td element